app/stacks/uk-west/common/key-vault.tf

```diff
@@ -67,3 +67,45 @@ resource "azurerm_key_vault_secret" "applications_service_vpn_gateway_shared_key
     ]
   }
 }
+
+resource "azurerm_key_vault_secret" "sql_server_password" {
```

Comment:
I don't think these secrets need to be in the common stack. They should live along side the database so this PR shouldn't need any resources in common.

Also, instead of a placeholder value, you can use the random provider in terraform to generate one - https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password

Reply:
Username could be something like `admin-{random id}`

Reply:
Resolved in https://github.com/Planning-Inspectorate/infrastructure-environments/pull/60/commits/c654cf28a9584b639ac7152fd10796f0d493b66e